src/button/Button.tsx

```diff
@@ -1,55 +1,33 @@
 import React from "react";
-import { Button as AriaButton } from "reakit";
 import classnames from "classnames";
+import { Button as AriaButton } from "reakit";
 
 import theme from "../theme";
 
 export interface ButtonProps {
-  /**
-   * Is this the principal call to action on the page?
-   */
-  primary?: boolean;
-  /**
-   * What background color to use
-   */
-  backgroundColor?: string;
   /**
    * How large should the button be?
    */
-  size?: "small" | "medium" | "large";
+  size?: "extra-small" | "small" | "medium" | "large";
   /**
    * Button contents
    */
-  label: string;
-  /**
-   * Optional click handler
-   */
-  onClick?: () => void;
+  children: string;
```

Comment:
children should not be typed implicitly. React.FC already has the correct definitions.